guides/common/modules/proc_configuring-a-host-for-registration.adoc

```diff
@@ -1,44 +1,34 @@
 [id="Configuring_a_Host_for_Registration_{context}"]
 = Configuring a Host for Registration
 
-{RHEL} virtual machines register to Customer Portal Subscription Management by default.
-You must update each virtual machine configuration so that they receive updates from the correct {Project} Server or {SmartProxyServer}.
+{EL} virtual machines register to Customer Portal Subscription Management by default.
+You must update each virtual machine configuration so that they receive updates from the correct {ProjectServer} or {SmartProxyServer}.
 
 .Prerequisites
-* Hosts must be using the following {EL} version:
-** 6.4 or later
-** 7.0 or later
-* All architectures of {EL} are supported (i386, x86_64, s390x, ppc_64).
-* Ensure that a time synchronization tool is enabled and runs on the {ProjectServer}s, any {SmartProxyServers}, and the hosts.
-** For {EL} 6:
-+
-----
-# chkconfig ntpd on; service ntpd start
-----
-** For {EL} 7:
+* The host must be using a supported {EL} version.
+For more information, see xref:supported-host-operating-systems-and-architectures-in-registration_{context}[].
+* Ensure that a time synchronization tool is enabled and runs on {ProjectServer} and any {SmartProxyServers}.
+
+.Procedure
+. Ensure that a time synchronization tool is enabled and running on the host.
+** For {EL} 7 and newer:
```

Comment:
Also depending on the OS of the managed host. OK for me for now, but I'll have to expand this. Maybe you can use https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L131 for now?

Reply:
What is it for? How do I use that? Where do I place it in the module?

Reply:
You can use it as a "normal" flag like "if foreman-deb". I'd place it below line 13 so that the bullet points for EL7+ etc are hidden for Debian and Ubuntu in downstream.

I realize that this is not perfect, but it's all we got right now.

Reply:
OK, but I have to say that the attribute is badly named :))

Reply:
Didn't you mean https://github.com/theforeman/foreman-documentation/blob/f13a8e1197adc7be3c9691e703a8b5b8c280922d/guides/common/attributes-base.adoc#L132 ?